src/openrct2/paint/Supports.h

```diff
@@ -17,8 +17,41 @@ struct FootpathPaintInfo;
 
 constexpr const uint8_t NumVanillaWoodenSupportTypes = 49;
 
+enum class WoodenSupportType : uint8_t
+{
+    Truss = 0,
+    Mine = 1,
+};
+
+enum class WoodenSupportSubType : uint8_t
+{
+    Dir0 = 0,
+    Dir1 = 1,
```

Comment:
isn't this a tad confusing since its direction 0,3 and 1,4. Could you either add a comment or adjust the name to work in that its for both ?true? directions.